controller/network/router_messaging.go

```diff
@@ -188,7 +188,12 @@ func (self *RouterMessaging) syncStates() {
 		}
 
 		if v.sendInProgress.Load() {
+			continue
+		}
 
+		if !notifyRouter.SupportsRouterLinkMgmt() {
+			// if the router doesn't support router based link mgmt, don't send these messages
+			delete(self.routerUpdates, k)
```

Comment:
seems like a log here would be useful? not every time but at least once in a while that this is happening? are there repercussions that somehow noticeable in some other way?